### DISCUSSION TITLE
No more support for Markdown ?

### DISCUSSION THREAD

**Discussion by JohnoCZ** [NamelessMC/Nameless#3423] [Q&A]
**Hi there,
I have a question if markdown is completely removed from nameless?** 

Because when I used nameless back in 2021, there was still the option to use markdown for forums which helps us with webhook to discord because we use the Czech language. And output on discord webhook looks like this:
![image](https://github.com/NamelessMC/Nameless/assets/55092714/90b514bd-cd63-418b-8ba0-944d33a7c2ce)

Now in 2023 I started using again nameless but without markdown our webhook to discord looks like this...
![image](https://github.com/NamelessMC/Nameless/assets/55092714/e487984e-e306-4c70-95db-182e38adf12f)

Its probably because Czech language uses symbols like ěščřžýáíé.... Is there any way how to fix this in webhooks?

**Comment by Derkades** [comment]
Html-encoded characters appearing in the webhook is a bug, are you using the latest NamelessMC version?

**Comment by JohnoCZ** [comment]
![image](https://github.com/NamelessMC/Nameless/assets/55092714/c9f3a320-a2b5-4e58-a11e-0a51820a3d8b)
Yes

**Reply by Derkades** [reply]
Thanks, I will investigate
